diff --git a/Tables/dbo.Book.sql b/Tables/dbo.Book.sql
index c825127..e86e1ab 100644
--- a/Tables/dbo.Book.sql
+++ b/Tables/dbo.Book.sql
@@ -2,7 +2,8 @@
 (
 [id] [int] NULL,
 [title] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
-[AuthorID] [int] NULL
+[AuthorID] [int] NULL,
+[ISBN] [int] NOT NULL
 ) ON [PRIMARY]
 ALTER TABLE [dbo].[Book] ADD
 CONSTRAINT [FK_Book_Author] FOREIGN KEY ([AuthorID]) REFERENCES [dbo].[Author] ([id]) ON DELETE CASCADE ON UPDATE CASCADE
